gui/wxpython/mapdisp/statusbar.py

```diff
@@ -182,35 +182,43 @@ def ShowStatusbarChoiceItemsByClass(self, itemClasses):
             if item.__class__ in itemClasses:
                 self.choice.Insert(item.label, pos, item)
 
+    def GetItemLabels(self):
+        """Get list of item labels"""
+        attributes = []
+        for value in self.statusbarItems.values():
+            if value.GetPosition() == 0:
+                attributes.append(value.label)
+        return attributes
```

Comment:
This is shorter, list comprehensions are typically preferred and once you get used to them, they are more readable:
```suggestion
        return [value.label for value in self.statusbarItems.values() if value.GetPosition() == 0]
```